scripts/mbedtls_dev/macro_collector.py

```diff
@@ -301,7 +302,7 @@ def read_file(self, header_file):
             self.read_line(line)
 
 
-class InputsForTest(PSAMacroCollector):
+class InputsForTest(PSAMacroEnumerator):
```

Comment:
Question that could help my review. This has been changed the other way around a few commit before (commit" Use InputsForTest in generate_psa_tests"). Not possible to avoid this back and forth ?

Reply:
I am not asking necessarily to change it but I don't really understand why you have to change this back and forth. Could you elaborate a bit on that ?